jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpanContext.java

```diff
@@ -166,14 +175,19 @@ boolean isDebugIdContainerOnly() {
    * debug trace, and the value of header recorded as a span tag to serve as a searchable
    * correlation ID.
    *
-   * @param debugId arbitrary string used as correlation ID
+   * @param debugId arbitrary string used as correlation ID.
+   * @param factory tracing object factory.
```

Comment:
The name TracingFactory was making me uncomfortable, I was thinking ObjectFactory, but it too was too generic. I think TracingObjectFactory is much cleaner name

Reply:
OK I struggled with the names. JaegerFactory or JaegerObjectFactory could work too.

Reply:
+1 JaegerObjectFactory